packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -311,14 +316,14 @@ export function finishMuting(
  * @param {Object} state Current state.
  * @param {Object} action Action object.
  * @param {Object} action.payload Action payload.
- * @param {string} action.payload.id Item ID.
+ * @param {string} action.payload.elementId Element ID.
  * @return {Object} New state
  */
-export function startTrimming(state, { payload: { id } }) {
+export function startTrimming(state, { payload: { elementId } }) {
```

Comment:
As per my other comment, we should not need to change this reducer.

Reply:
reverted here - https://github.com/GoogleForCreators/web-stories-wp/pull/10712/commits/144962ae691692e2ae1f05db9ea8dd408c925b50